franklin/users/templates/dashboard.html

```diff
@@ -3,5 +3,14 @@
 {% block main %}
 <div>
     <h1 class="">Welcome {{ request.user.get_username }}!</h1>
+    {% if repos %}
+    {% for repo in repos %}
+    <div class="">
+        <a class="" href="{{ repo.url }}">
+            {{ repo.name }}
+        </a>
+    </div>
+    {% endfor %}
+    {% endif %}
```

Comment:
This is the real value. Run the app and log in with github to see this working. Next PR will add a `deploy this app` button for each repo.